docs/extend/menu.md

```diff
@@ -1,22 +1,20 @@
 <!--[metadata]>
 +++
-title = "Extend Engine"
+title = "Extend Docker"
 description = "How to extend Docker Engine with plugins"
 keywords = ["extend, plugins, docker, documentation, developer"]
+type="menu"
 [menu.main]
 identifier = "engine_extend"
-parent = "engine_use"
-weight = 6
+parent="engine_use"
+weight = 0
 +++
 <![end-metadata]-->
 
 
-## Extending Docker Engine
+## New Docker Plugin System
```

Comment:
I'm a bit wary of using "New", because it's new *now* but soon that's outdated. Not sure what a good name would be ("managed plugins"?)